system/ee/ExpressionEngine/Addons/structure/views/index.php

```diff
@@ -145,10 +145,19 @@
                     }
                 }
 
-                if ($page['hidden'] == 'y') {
+                if (isset($settings['show_entry_status']) && $settings['show_entry_status'] == 'y') {
+                    $status = ee('Model')->get('Status')->filter('status', '==', $page['status'])->first();
+                    echo $status->renderTag();
+                }
+
+                if (isset($settings['show_nav_status']) && $settings['show_nav_status'] == 'y' && $page['hidden'] == 'y') {
                     echo ' <span class="hidden-page">(hidden)</span>';
                 }
 
+                if (isset($page['extra_html']) && $page['extra_html']) {
```

Comment:
Added this as a way to use the `structure_index_view_data` hook to add even further UI elements as needed. For instance, additional Publisher tags:

<img width="416" alt="image" src="https://github.com/ExpressionEngine/ExpressionEngine/assets/2423727/6e8b5077-06fc-409e-a1b3-12c3a53f56b8">